### DISCUSSION TITLE
Duda al subir archivos de html

### DISCUSSION THREAD

**Discussion by ahranza** [LaunchX-InnovaccionVirtual/FrontEnd-Mision#211] [Q&A]
Hola! Lo archivos que tenemos que subir de repositorios de la pastelería son las paginas? O el código de Visual Studio? 
Gracias!

**Comment by JuanSalvador73** [comment]
Principalmente solo el HTML, el código. Si tienes mas que eso mejor. Y las imágenes, si tienes por supuesto.

**Reply by ahranza** [reply]
Muchas Gracias.
Cuando quiero arrastrar los archivos al repositorio sólo me salen las páginas. No sé como subir los códigos. Es que apenas estoy prendiendo desde cero.

**Reply by hackit92** [reply]
@ahranza Estás usando visual studio code? De ser así conectalo con tu repositorio y subelos desde ahi, sino es el caso ayudanos con una captura para entender bien el problema. =)

**Comment by doguedogue** [comment]
En tu repositorio de github subes los archivos .html que hayas creado

1. Desde github ubicate en tu repositorio y selecciona 
![image](https://user-images.githubusercontent.com/23409026/156512527-ee3a221a-1d24-4e2d-90b3-09a07879a985.png)

3. En la parte inferior pones commit y ya

**Reply by ahranza** [reply]
Muchas Gracias!

**Comment by ahranza** [comment]
Esos archivos así los subo al repositorio? Sí, lo hice en Visual.

![2022-03-03 (2)](https://user-images.githubusercontent.com/99292356/156515412-8c8a73f2-9c43-4146-8d93-ff6d64560798.png)

**Reply by hackit92** [reply]
Como bien te sugiere dogue, puedes subir manualmente tus archivos al repositorio, mi sugerencia era en este apartado de visual studio code, para actualizar tu repo desde ahí añadiendo comentarios, eso te ayuda a tener un poco más de control de una forma más rápida, pero ambas te funcionan =)
<img width="558" alt="Captura de Pantalla 2022-03-03 a la(s) 1 23 42" src="https://user-images.githubusercontent.com/39970401/156516397-095d5b7f-9130-4d68-a33f-8d54fc5ece9a.png">

**Reply by ahranza** [reply]
Muchas Gracias!

**Reply by carlos-gmx3** [reply]
Que tema estas usando en VS Code?
